specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2025-03-01-preview/apimgatewayHostnameBindings.json

```diff
@@ -0,0 +1,425 @@
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/gateways/{gatewayName}/hostnameBindings": {
+      "get": {
+        "tags": [
+          "ApiGatewayHostnameBinding"
+        ],
+        "operationId": "ApiGatewayHostnameBinding_ListByGateway",
+        "description": "List all API Management gateway hostname bindings within a gateway.",
+        "x-ms-examples": {
+          "ApiManagementListGatewayHostnameBindings": {
+            "$ref": "./examples/ApiManagementListGatewayHostnameBindings.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/GatewayNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The API Management gateway hostname binding list.",
+            "schema": {
+              "$ref": "#/definitions/GatewayHostnameBindingListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/gateways/{gatewayName}/hostnameBindings/{hostnameBindingName}": {
+      "get": {
+        "tags": [
+          "ApiGatewayHostnameBinding"
+        ],
+        "operationId": "ApiGatewayHostnameBinding_Get",
+        "description": "Gets an API Management gateway hostname binding resource description.",
+        "x-ms-examples": {
+          "ApiManagementGetGatewayHostnameBinding": {
+            "$ref": "./examples/ApiManagementGetGatewayHostnameBinding.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/GatewayNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/HostnameBindingNameParameter"
```

Comment:
This is `GatewayHostnameBindingNameParameter`

Reply:
Changed to GatewayHostnameBindingNameParameter